tests/Parsing/TopLevelCommentTest.elm

```diff
@@ -0,0 +1,238 @@
+module Parsing.TopLevelCommentTest exposing (parseTopLevelCommentTest)
+
+import Elm.Parser
+import Elm.Processing
+import Elm.Syntax.File
+import Elm.Syntax.Range
+import Expect
+import Intermediate
+import Models
+import Test
+import TestUtil
+
```

Comment:
Please see the previous comments related to the tests and don't forget to apply them here.